src/App.tsx

```diff
@@ -1,4 +1,7 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable no-console */
```

Comment:
Try to fix eslint rules and remove comments

Reply:
Not resolved. Try to remove comments. If you have any questions - feel free to ask them in `fe_chat`